### PR TITLE
Config mgmt recipe

### PR DESCRIPTION
**Pull request by Loquacity**
~***DO NOT MERGE. DOCS REPO IS FROZEN***~

# Description

Draft of recipe for Configuration Management, plus a couple of small drive-by edits on associated pages.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Partial https://github.com/SUSE/spacewalk/issues/14304
Epic https://github.com/SUSE/spacewalk/issues/12021

### CODE REVIEW

File: modules/cookbook/pages/recipe-configmgmt.adoc [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+
+
+== Before you begin
+
+Before you start, you should already have:
+
+* A {productname} Server installed, that you can access using the {webui}.
+* At least one Salt client registered to your server.
+* Appropriate subscriptions from http://scc.suse.com for the products you are using.
```
Comment (keichwa) [inline modules/cookbook/pages/recipe-configmgmt.adoc:29]: I think this is optional (Uyuni).

File: modules/cookbook/pages/recipe-configmgmt.adoc [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+
+
+== Steps
+
+Create a new configuration channel and file::
+
+. In the {productname} {webui}, navigate to menu:Configuration[Channels] and click btn:[Create State Channel].
+. Type a name, label, and description for your configuration file, and type the contents of your configuration file.
+A simple example that you can copy is at the end of this section.
```
Comment (keichwa) [inline modules/cookbook/pages/recipe-configmgmt.adoc:43]:
```suggestion
An example that you can copy is at the end of this section.
```

File: modules/cookbook/pages/recipe-configmgmt.adoc [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+= Configuration Management
+
+You can use configuration files and channels to manage configuration for your clients, rather than configuring each client manually.
+This recipe shows you how to use the {productname} {webui} to create a centrally-managed configuration file, assign it to a Salt client, and apply the configuration.
+
+
+
+== Problem
+
+If you are managing a lot of clients, you probably don't want to manually apply configuration settings to each of them in turn.
```
Comment (keichwa) [inline modules/cookbook/pages/recipe-configmgmt.adoc:11]:
```suggestion
If you are managing a lot of clients, you probably do not want to manually apply configuration settings to each of them in turn.
```

File: modules/cookbook/pages/recipe-configmgmt.adoc [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+For more information about the different ways to manage configuration, see xref:client-configuration:configuration-management.adoc[].
+
+
+
+== Steps
+
+Create a new configuration channel and file::
+
+. In the {productname} {webui}, navigate to menu:Configuration[Channels] and click btn:[Create State Channel].
+. Type a name, label, and description for your configuration file, and type the contents of your configuration file.
```
Comment (keichwa) [inline modules/cookbook/pages/recipe-configmgmt.adoc:42]: I think you first want to enter the details of the channel, the cfg file will come later.

File: modules/cookbook/pages/recipe-configmgmt.adoc [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+Assign clients to the configuration channel::
+
+. In the {productname} {webui}, navigate to menu:Systems[Systems List] and select the client you want to assign to your configuration channel.
+. Navigate to the [guimenu]``Configuration`` tab.
+  In the guimenu:``Configuration Overview`` section, click btn:[Subscribe to channels].
+. Check the configuration channel you created earlier, and click btn:[Continue].
++
+image::assign_client_cfg.png[scaledwidth=80%]
+. If you have more than one configuration channel, you can rank them in order of importance, or just click btn:[Update Channel Rankings] to finish.
```
Comment (keichwa) [inline modules/cookbook/pages/recipe-configmgmt.adoc:56]:
```suggestion
. If you have more than one configuration channel, you can rank them in order of importance, or click btn:[Update Channel Rankings] to finish.
```

**Review by keichwa — CHANGES_REQUESTED** [review]
Wondering whether we want to use sentence style for section titles and other title throughout this book?

**Review by jcayouette — APPROVED** [review]
Looks good other than requested changes.

File: modules/cookbook/pages/recipe-configmgmt.adoc (hunk 1 above)
Comment (Loquacity) [reply modules/cookbook/pages/recipe-configmgmt.adoc:29]: True, although there's a lot of caveats in that sentence ("appropriate" etc)

File: modules/cookbook/pages/recipe-configmgmt.adoc (hunk 4 above)
Comment (Loquacity) [reply modules/cookbook/pages/recipe-configmgmt.adoc:42]: Not really, it's a very weird UI.

**Review by keichwa — APPROVED** [review]

**Pull request merged**